distribution/lib/Standard/Visualization/0.0.0-dev/src/Table/Visualization.enso

```diff
@@ -59,20 +59,42 @@ prepare_visualization y max_rows=1000 =
    Column Limit
 max_columns = 250
 
+##PRIVATE
+nothing_count : Column -> Integer
```

Comment:
Doesn't Column already have count_nothing? Or is this different?

Reply:
I didn't know! If it's already got the method I'll use that!

Reply:
If you also add count_whitespace to Column, please add a test to Table_Spec in the "Counting Values" section (and a "not supported" version to DB_Column).